app/pubspec.yaml

```diff
@@ -48,6 +48,7 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   flutter_launcher_icons: ^0.9.2
+  fluttertoast: ^8.0.9
```

Comment:
why introduce another external component, when material design [already has a snackbar component](https://material.io/components/snackbars)? Unless there is an important feature that provides (and we are using), let's stick with the material already included components. Please replace the flutter toast with the material snackbar.